### PR TITLE
Transition to Rig 1.0.0

### PR DESCRIPTION
**Pull request by mundya**
Moves applications to SpiNNaker tools 2.0.1 and Rig over to 1.0.0!

### CODE REVIEW

**Comment by mundya** [comment]
Durr, I'll fix the tests.

File: nengo_spinnaker/simulator.py [hunk 1]
```diff
@@ -171,12 +169,10 @@ def __init__(self, network, dt=0.001, period=10.0, timescale=1.0):
         ))
 
         logger.info("Setting router timeout to 16 cycles")
-        for x in range(machine_width):
-            for y in range(machine_height):
-                with self.controller(x=x, y=y):
-                    if (x, y) in system_info:
-                        data = self.controller.read(0xf1000000, 4)
-                        self.controller.write(0xf1000000, data[:-1] + b'\x10')
+        for x, y in system_info.chips():
```
Comment (mossblaser) [inline nengo_spinnaker/simulator.py:172]: Just iterating over `system_info` is sufficient. (It is a dictionary {(x, y): chip_info}).

File: spinnaker_components/README.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 Code for SpiNNaker executables
 ==============================
 
-The SpiNNaker executables require `spinnaker_tools` v1.3.3 as released by Steve
-Temple to build.  If this is available then, after sourcing the `setup` file in
-the `spinnaker_tools` directory and making `sark` and `spin1_api` the Nengo
+The SpiNNaker executables require [`spinnaker_tools`
+v2.0.1](http://apt.cs.manchester.ac.uk/projects/SpiNNaker/downloads/) to build.
+If this is available then, after sourcing the `setup` file in the
+`spinnaker_tools` directory and making `sark` and `spin1_api` the Nengo
```
Comment (mossblaser) [inline spinnaker_components/README.md]: Comma missing after `spin1_api` ?

**Comment by mossblaser** [comment]
I haven't looked at things not touched by this commit but this looks entirely sensible to me!

File: nengo_spinnaker/simulator.py (hunk 1 above)
Comment (mundya) [inline nengo_spinnaker/simulator.py:172]: I don't think it's as clear; `for x, y in machine` was OK but it's not immediately clear that a `sys_info` contains co-ordinates.

**Pull request merged**